model/stats/db.go

```diff
@@ -601,6 +601,28 @@ func (filter StatsFilter) buildMatchStageForTest() bson.M {
 		DbTestStatsIdRequesterKeyFull: bson.M{"$in": filter.Requesters},
 	}
 
+	// After cutting over the a few build variants to use resmoke spawned
+	// with jasper (SERVER-54315), test names are now random UUIDs with
+	// with a human-readble display name. For the period between
+	// 09/01/21-09/14/21, the historical test stats calculations failed to
+	// account for this (EVG-15396). We should ignore any test stats
+	// affected by this bug.
+	//
+	// TODO: (EVG-15396) Remove this code once all affected test stats TTL
```

Comment:
Ideally, we would remember to do this in March. Since EVG-15396 is this ticket, are you going to make a separate ticket to delete this code or are you going to put EVG-15396 on the backlog until the TTL is hit?

Reply:
Created https://jira.mongodb.org/browse/EVG-15408